### DISCUSSION TITLE
Attribute Support delegation

### DISCUSSION THREAD

**Discussion by azbo** [dotnet/csharplang#5932] [Language Ideas] [answered]
[AttributeUsage(AttributeTargets.Method)]
    public class QueryAttribute : Attribute
    {
        public QueryAttribute(Func<int,bool> v)
        {
        }
    }

public interface IUserService
{

    [Query(t => t>10)]
    public UserInfo GetUserInfo(int id, string name);
}

**Comment by Phantonia** [comment]
Could you please explain what that would do?

**Reply by FaustVX** [reply]
I think he want to be able to store delegates in `Attribute`

**Comment by Joe4evr** [comment, marked as answer]
Duplicate of #343
